qa-niaid.planx-pla.net/manifest.json

```diff
@@ -16,7 +16,7 @@
     "pidgin": "quay.io/cdis/pidgin:2020.08",
     "revproxy": "quay.io/cdis/nginx:2020.08",
     "sheepdog": "quay.io/cdis/sheepdog:2020.08",
-    "portal": "quay.io/cdis/data-portal:nct-1.0.0",
+    "portal": "quay.io/cdis/data-portal:feat_niaid-ct",
```

Comment:
did you get your branch into this one yet?

Reply:
no, i'm just getting ready for when i do 😆